centaur/src/main/resources/standardTestCases/papi_preemptible_and_max_retries.test

```diff
@@ -0,0 +1,12 @@
+name: papi_preemptible_and_max_retries
+testFormat: workflowfailure
```

Comment:
TOL: is it possible to check that the appropriate call made three attempts before finally failing? I'm not 100% sure but there might be an existing centaur test option to check for that